sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormFieldType.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// </summary>
+    [CodeGenModel("FieldValueType")]
+    public enum FieldValueType
```

Comment:
Do we expect this to change? should we make them extensible enums?

Reply:
I chatted with @KrzysztofCwalina about this a bit.  I understand that extensible enums are used where values may change between SDK releases.  We'd expect any new field types to correspond to a new SDK release, and if they didn't, callers can get at the value as a string and cast it themselves.

Reply:
Right, so should we convert it to an extensible enum? not sure how this works with codegen, but we can take a look

Reply:
When values will only change with a new SDK release, we can use a regular enum.  I don't think these values will change _between_ releases.

Reply:
Got it. Thanks Anne!